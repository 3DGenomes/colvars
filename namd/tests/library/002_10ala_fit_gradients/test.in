colvarsTrajFrequency    1

indexFile ../Common/da.ndx


colvar {
    name eigenvector_Ca

    outputTotalForce on
    outputAppliedForce on
    width 1.0 

    eigenvector {
        #    debugGradients
        atoms {
            enableFitGradients yes
            indexGroup Protein_C-alpha
        }
        refpositionsfile ../Common/pi-ideal.xyz
        #    vector { (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) }
        vectorFile ../Common/da.xyz
        differenceVector yes
    }
} 


harmonic {
    colvars        eigenvector_Ca
    forceConstant  10.0  
    centers        0.0
}
