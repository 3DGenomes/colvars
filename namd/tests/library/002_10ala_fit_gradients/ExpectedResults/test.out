Charm++: standalone mode (not using charmrun)
Converse/Charm++ Commit ID: v6.4.0-beta1-0-g5776d21
CharmLB> Load balancer assumes all CPUs are same.
Charm++> Running on 1 unique compute nodes (6-way SMP).
Charm++> cpu topology info is gathered in 0.000 seconds.
Info: NAMD 2.9 for Linux-x86_64-multicore
Info: 
Info: Please visit http://www.ks.uiuc.edu/Research/namd/
Info: for updates, documentation, and support information.
Info: 
Info: Please cite Phillips et al., J. Comp. Chem. 26:1781-1802 (2005)
Info: in all publications reporting results obtained with NAMD.
Info: 
Info: Based on Charm++/Converse 60400 for multicore-linux64
Info: Built Thu Jun 13 17:22:49 EDT 2013 by gfiorin on helicon
Info: 1 NAMD  2.9  Linux-x86_64-multicore  1    helicon  gfiorin
Info: Running on 1 processors, 1 nodes, 1 physical nodes.
Info: CPU topology information available.
Info: Charm++/Converse parallel runtime startup completed at 0.00233102 s
Info: 40.8164 MB of memory in use based on /proc/self/stat
Info: Configuration file is test.namd
Info: Working in the current directory /home/gfiorin/sources/colvars/namd/tests/library/002_10ala_fit_gradients
TCL: Suspending until startup complete.
Info: SIMULATION PARAMETERS:
Info: TIMESTEP               0.5
Info: NUMBER OF STEPS        0
Info: STEPS PER CYCLE        20
Info: LOAD BALANCER  Centralized
Info: LOAD BALANCING STRATEGY  New Load Balancers -- DEFAULT
Info: LDB PERIOD             4000 steps
Info: FIRST LDB TIMESTEP     100
Info: LAST LDB TIMESTEP     -1
Info: LDB BACKGROUND SCALING 1
Info: HOM BACKGROUND SCALING 1
Info: MIN ATOMS PER PATCH    40
Info: INITIAL TEMPERATURE    250
Info: CENTER OF MASS MOVING INITIALLY? NO
Info: DIELECTRIC             1
Info: EXCLUDE                SCALED ONE-FOUR
Info: 1-4 ELECTROSTATICS SCALED BY 1
Info: MODIFIED 1-4 VDW PARAMETERS WILL BE USED
Info: DCD FILENAME           test.dcd
Info: DCD FREQUENCY          2000
Info: DCD FIRST STEP         2000
Info: NO EXTENDED SYSTEM TRAJECTORY OUTPUT
Info: NO VELOCITY DCD OUTPUT
Info: NO FORCE DCD OUTPUT
Info: OUTPUT FILENAME        test
Info: NO RESTART FILE
Info: SWITCHING ACTIVE
Info: SWITCHING ON           9
Info: SWITCHING OFF          11
Info: PAIRLIST DISTANCE      12
Info: PAIRLIST SHRINK RATE   0.01
Info: PAIRLIST GROW RATE     0.01
Info: PAIRLIST TRIGGER       0.3
Info: PAIRLISTS PER CYCLE    2
Info: PAIRLISTS ENABLED
Info: MARGIN                 1
Info: HYDROGEN GROUP CUTOFF  2.8
Info: PATCH DIMENSION        15.8
Info: ENERGY OUTPUT STEPS    100
Info: CROSSTERM ENERGY INCLUDED IN DIHEDRAL
Info: TIMING OUTPUT STEPS    2000
Info: COLLECTIVE VARIABLES CALCULATION REQUESTED
Info: COLLECTIVE VARIABLES CONFIGURATION   test.in
Info: LANGEVIN DYNAMICS ACTIVE
Info: LANGEVIN TEMPERATURE   300
Info: LANGEVIN DAMPING COEFFICIENT IS 10 INVERSE PS
Info: LANGEVIN DYNAMICS APPLIED TO HYDROGENS
Info: USING VERLET I (r-RESPA) MTS SCHEME.
Info: C1 SPLITTING OF LONG RANGE ELECTROSTATICS
Info: PLACING ATOMS IN PATCHES BY HYDROGEN GROUPS
Info: RANDOM NUMBER SEED     87654321
Info: USE HYDROGEN BONDS?    NO
Info: COORDINATE PDB         ../Common/da.pdb
Info: STRUCTURE FILE         ../Common/da.psf
Info: PARAMETER file: CHARMM format! 
Info: PARAMETERS             ../Common/par_all22_prot.inp
Info: USING ARITHMETIC MEAN TO COMBINE L-J SIGMA PARAMETERS
Info: SUMMARY OF PARAMETERS:
Info: 140 BONDS
Info: 356 ANGLES
Info: 459 DIHEDRAL
Info: 43 IMPROPER
Info: 6 CROSSTERM
Info: 95 VDW
Info: 0 VDW_PAIRS
Info: 0 NBTHOLE_PAIRS
Info: TIME FOR READING PSF FILE: 0.00157595
Info: TIME FOR READING PDB FILE: 0.000159979
Info: 
Info: ****************************
Info: STRUCTURE SUMMARY:
Info: 104 ATOMS
Info: 103 BONDS
Info: 183 ANGLES
Info: 251 DIHEDRALS
Info: 22 IMPROPERS
Info: 0 CROSSTERMS
Info: 0 EXCLUSIONS
Info: 312 DEGREES OF FREEDOM
Info: 51 HYDROGEN GROUPS
Info: 4 ATOMS IN LARGEST HYDROGEN GROUP
Info: 51 MIGRATION GROUPS
Info: 4 ATOMS IN LARGEST MIGRATION GROUP
Info: TOTAL MASS = 727.821 amu
Info: TOTAL CHARGE = 6.70552e-08 e
Info: *****************************
Info: 
Info: Entering startup at 0.01723 s, 56.4141 MB of memory in use
Info: Startup phase 0 took 4.19617e-05 s, 56.4141 MB of memory in use
Info: Startup phase 1 took 0.000173092 s, 56.4141 MB of memory in use
Info: Startup phase 2 took 5.19753e-05 s, 56.4141 MB of memory in use
Info: Startup phase 3 took 3.19481e-05 s, 56.4141 MB of memory in use
Info: PATCH GRID IS 2 BY 1 BY 1
Info: PATCH GRID IS 1-AWAY BY 1-AWAY BY 1-AWAY
Info: REMOVING COM VELOCITY 0.177776 1.11835 0.497162
Info: LARGEST PATCH (1) HAS 54 ATOMS
Info: Startup phase 4 took 0.000137091 s, 56.5781 MB of memory in use
Info: Startup phase 5 took 3.60012e-05 s, 56.5781 MB of memory in use
Info: Startup phase 6 took 3.00407e-05 s, 56.5781 MB of memory in use
LDB: Central LB being created...
Info: Startup phase 7 took 3.98159e-05 s, 56.5781 MB of memory in use
Info: CREATING 19 COMPUTE OBJECTS
Info: NONBONDED TABLE R-SQUARED SPACING: 0.0625
Info: NONBONDED TABLE SIZE: 705 POINTS
Info: ABSOLUTE IMPRECISION IN VDWB TABLE ENERGY: 1.03398e-25 AT 10.9972
Info: RELATIVE IMPRECISION IN VDWB TABLE ENERGY: 1.12007e-16 AT 10.9972
colvars: ----------------------------------------------------------------------
colvars: Initializing the collective variables module, version 2013-06-13.
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = eigenvector_Ca
colvars:   Initializing a new "eigenvector" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/pi-ideal.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating the vector projection: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     # vector =  [default]
colvars:     # vectorFile = ../Common/da.pdb
colvars:     # vectorCol = "" [default]
colvars:     Geometric center of the provided vector: ( 0.9999 , 2 , 3 )
colvars:     # differenceVector = on
colvars:     "differenceVector" is on: subtracting the reference positions from the provided vector: v = v - x0.
colvars:     "differenceVector" is on: normalizing the vector.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0 [default]
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # upperWallConstant = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { eigenvector_Ca }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 10
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars:   # outputAccumulatedWork = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module initialized.
colvars: ----------------------------------------------------------------------
Info: Startup phase 8 took 0.00303912 s, 57.1797 MB of memory in use
Info: Startup phase 9 took 4.29153e-05 s, 57.1797 MB of memory in use
Info: Startup phase 10 took 4.1008e-05 s, 57.5156 MB of memory in use
Info: Finished startup at 0.020895 s, 57.5156 MB of memory in use

colvars: Saving collective variables state to "test.colvars.state".
ETITLE:      TS           BOND          ANGLE          DIHED          IMPRP               ELECT            VDW       BOUNDARY           MISC        KINETIC               TOTAL           TEMP      POTENTIAL         TOTAL3        TEMPAVG

ENERGY:       0        17.7452        62.9761        30.7071         8.3800             62.2194       -13.6978         0.0000         5.0000        69.9033            243.2333       225.4931       173.3300       243.2948       225.4931

WRITING EXTENDED SYSTEM TO OUTPUT FILE AT STEP 0
WRITING COORDINATES TO OUTPUT FILE AT STEP 0
COORDINATE DCD FILE WAS NOT CREATED
The last position output (seq=-2) takes 0.000 seconds, 57.645 MB of memory in use
WRITING VELOCITIES TO OUTPUT FILE AT STEP 0
The last velocity output (seq=-2) takes 0.000 seconds, 57.645 MB of memory in use
====================================================

WallClock: 0.021682  CPUTime: 0.020001  Memory: 57.644531 MB
Program finished.
